sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/FormContent.java

```diff
@@ -12,7 +12,7 @@
 public abstract class FormContent {
 
     /*
-     * Text content of the extracted field.
+     * Text content of the extracted line.
```

Comment:
consider changing to `extracted element`. it could be a whole line or a single word and eventually it could be the selection in a checkbox element